tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -578,10 +578,7 @@ public function triggerAsyncUpload(string $enabledOrDisabled):void {
 			$value = 'false';
 		}
 		if ($this->previousAsyncSetting === null) {
-			$previousAsyncSetting = SetupHelper::runOcc(
-				['config:system:get', 'dav.enable.async'],
-				$this->getStepLineRef()
-			)['stdOut'];
+			$previousAsyncSetting = ['code' => '', 'stdOut' => '', 'stdErr' => '' ]['stdOut'];
 			$this->previousAsyncSetting = \trim($previousAsyncSetting);
```

Comment:
line 587 can be removed too?

Reply:
line 587 `runOcc` calls method `runOcc` in FeatureContext, which will be removed in the next PR.